lib/helpers/webview.ts

```diff
@@ -396,12 +396,18 @@ const WebviewHelpers: WebviewHelpers = {
 
     const result: string[] = [];
     for (const {webview, pages, proc, webviewName} of webviewsMapping) {
-      if (ensureWebviewsHavePages && pages?.length === 0) {
```

Comment:
Simply change `pages?.length === 0` to `!pages?.length`

Reply:
Is it mentioned about the diff *after* the change `pages.length === 0`?